### DISCUSSION TITLE
Additional Tag functions

### DISCUSSION THREAD

**Discussion by pooldoo** [vincode-io/Zavala#133] [Ideas]
It appears, on the iPad version at least, sorting by tags can only be done on a single tag at a time. Being able to select multiple tags, as in the new apple notes app would be a major help.

Even better would be the ability to add Boolean connectors to your filter. “(TagA or TagB) but not TagC. 

And if we could name those searches and assign them to buttons in the tag area… That would be a wonderful thing.

**Comment by vincode-io** [comment]
Thanks for the feedback. I'll add multiple select for tags into the 2.0 release that is currently in development.

I do have plans for advanced search options like you are suggesting, but those will probably come out in the 3.0 release. You can comment on #64 if you think of anything else.
